cmd/magneticow/api.go

```diff
@@ -50,15 +58,15 @@ func NewApiReadmeHandler() (*ApiReadmeHandler, error) {
 
 func (h *ApiReadmeHandler) Close() {
 	h.client.Close()
-	_ = os.RemoveAll(h.tempdir)
+	_ = os.RemoveAll(h.tempDir)
 }
 
 func (h *ApiReadmeHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	infohashHex := mux.Vars(r)["infohash"]
 
 	infohash, err := hex.DecodeString(infohashHex)
 	if err != nil {
-		respondError(w, http.StatusBadRequest, "couldn't decode infohash: %s", err.Error())
+		respondError(w, http.StatusBadRequest, MsgCantDecode)
```

Comment:
We could keep the error message. Usually, `DecodeString` returns an `InvalidByteError` in case of issues, which might be helpful information during troubleshooting.